.gitignore

```diff
@@ -18,6 +18,7 @@
 /Library/PinnedKegs
 /Library/PinnedTaps
 /Library/Taps
+/Library/lock.d
```

Comment:
Yeah, would make more sense. One minor thing: it should be a per-repo lock, not a system-wide one, if we're being sticklers for correctness.

I could see this actually being a use case, bc if I were running multiple brew installations for testing; I'd sometimes want to fire off an update for all of them, and would be convenient if it were safe to do that in parallel. Including a lunging of the repo path (and maybe host) in the file name of the actual lock directory would be sufficient.

Could also go on /var/lock, which I think is conventional for this sort of thing.